src/control/lib/daos/system_prop.go

```diff
@@ -516,10 +533,6 @@ func pph2sp(key SystemPropertyKey, pph *PoolPropHandler, def string) SystemPrope
 // SystemPropertySelfHealHasFlag returns true if the given self-heal property
 // value contains the specified flag.
 func SystemPropertySelfHealHasFlag(value string, flag SelfHealFlag) bool {
-	if value == "none" {
-		return false
-	}
-
```

Comment:
[Question] Why is this no longer necessary?

Reply:
because it's covered by the below block anyway, "none" won't match any valid flags and false will be returned